tests/data/dsl-semantic-validation-cases.yaml

```diff
@@ -1088,6 +1088,27 @@
         symbol: "account"
         errorType: duplicated-error
 ## Conditions
+- name: condition not defined
+  dsl: |
+    model
+      schema 1.1
+
+    type user
+
+    type document
+      relations
+        define viewer: [user with c]
+  expected_errors:
+    - msg: "`c` is not a defined condition in the model."
+      line:
+        start: 8
+        end: 8
+      column:
+        start: 31
```

Comment:
Are we starting from index 1 elsewhere? From index 0, this should be 30->31

Reply:
Agreed to standardize moving forward.